src/Type/Constant/ConstantArrayType.php

```diff
@@ -1162,16 +1179,10 @@ public function slice(int $offset, ?int $limit, bool $preserveKeys = false): sel
 		return $preserveKeys ? $slice : $slice->reindex();
 	}
 
+	/** @deprecated Use reverseArray() instead */
```

Comment:
Does it make sense to deprecate the method only for the ConstantArrayType?

Reply:
Sorry, I didn't fully get that. The method only exists there and nowhere else.

Reply:
Ohh I see ;-)